privacy.html

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Privacy Policy
 ---
-
+<script>function gaOptout() {document.cookie = disableStr + '=true; expires=Thu, 31 Dec 2099 23:59:59 UTC; path=/';window[disableStr] = true;}</script>

```

Comment:
I have just taken the standard coding. But an alert box would be fine. I make a note and will look at it later.